.github/actions/mssql/action.yml

```diff
@@ -27,7 +27,7 @@ inputs:
   log:
     description: 'Enable MSSQL database logging'
     required: false
-    default: false
+    default: 'false'
```

Comment:
does it have to be a string?

Reply:
Yes as default values can only be string.
https://docs.github.com/en/actions/sharing-automations/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault